.travis.yml

```diff
@@ -46,10 +46,30 @@ jobs:
 
     - stage: publish images
       if: type = push AND (tag IS present OR branch = master)
+      env:
+        - CHANGE_MINIKUBE_NONE_USER=true
       before_script:
+        # Decrypt credentials needed to log into gcr registry
         - openssl aes-256-cbc -K $encrypted_b48f9e852489_key -iv $encrypted_b48f9e852489_iv -in .travis/spire-travis-ci.json.enc -out .travis/spire-travis-ci.json -d
+        # Download stable kubectl
+        - curl -s -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
+        # Download latest minikube
+        - curl -s -Lo minikube https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64 && chmod +x minikube && sudo mv minikube /usr/local/bin/
+        # Start up minikube
+        - sudo minikube start --vm-driver=none
+        # Make sure kubectl configuration is up to date
+        - sudo chown -R $USER.$USER ~/.kube
+        - sudo chown -R $USER.$USER ~/.minikube
+        - minikube update-context
+        # Wait for stuff to become ready
+        - JSONPATH='{range .items[*]}{@.metadata.name}:{range @.status.conditions[*]}{@.type}={@.status};{end}{end}'; until kubectl get nodes -o jsonpath="$JSONPATH" 2>&1 | grep -q "Ready=True"; do sleep 1; done
```

Comment:
If this keeps running forever, it will be handled by a travis timeout, correct?

Reply:
Yes, that is correct.